src/main/java/com/example/NiRi/NiRiApplication.java

```diff
@@ -3,40 +3,37 @@
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
 
 
 @SpringBootApplication
+@ComponentScan(basePackages = "com.example.NiRi")
```

Comment:
this is not needed in spring boot. The component scanning is done automatically

Reply:
I´ve removed it. Thnk you.